data_process/get_stats_fv3gfs.py

```diff
@@ -10,9 +10,12 @@
 with xr.set_options(keep_attrs=True):
     global_means = ds.mean(dim=['time', 'grid_xt', 'grid_yt'])
     global_stds = ds.std(dim=['time', 'grid_xt', 'grid_yt'])
+    time_means = ds.mean(dim='time')
 
 global_means.to_netcdf(os.path.join(OUTPUT_PATH, 'fv3gfs-mean.nc'))
 global_stds.to_netcdf(os.path.join(OUTPUT_PATH, 'fv3gfs-stddev.nc'))
+time_means.to_netcdf(os.path.join(OUTPUT_PATH, 'fv3gfs-time-mean.nc'))
 
-print("means: ", global_means)
+print("global means: ", global_means)
```

Comment:
`global_means` is a single float (averaged over space and time), right?

Reply:
It's a single float for each variable, yes. The normalization uses a single mean and standard deviation for each variable, which are computed over space and time as you say. The "time mean" statistic, which does depend on lat/lon, is used for validation only, for computing the ACC.